int32 angle
---
int32 result
